### PR TITLE
Update storage driver options link

### PR DESCRIPTION
**Pull request by tonistiigi**
Old link redirects but loses the anchor.

@thaJeztah 

Signed-off-by: Tonis Tiigi <tonistiigi@gmail.com>

### CODE REVIEW

File: daemon/graphdriver/devmapper/deviceset.go [hunk 1]
```diff
@@ -1706,9 +1706,9 @@ func (devices *DeviceSet) initDevmapper(doInit bool) error {
 	// https://github.com/docker/docker/issues/4036
 	if supported := devicemapper.UdevSetSyncSupport(true); !supported {
 		if dockerversion.IAmStatic == "true" {
-			logrus.Error("devmapper: Udev sync is not supported. This will lead to data loss and unexpected behavior. Install a dynamic binary to use devicemapper or select a different storage driver. For more information, see https://docs.docker.com/engine/reference/commandline/daemon/#daemon-storage-driver-option")
+			logrus.Error("devmapper: Udev sync is not supported. This will lead to data loss and unexpected behavior. Install a dynamic binary to use devicemapper or select a different storage driver. For more information, see https://docs.docker.com/engine/reference/commandline/dockerd/#/storage-driver-options")
```
Comment (thaJeztah) [inline daemon/graphdriver/devmapper/deviceset.go]: can you change the anchor and remove the extra slash (`/#/storage-driver-options` -> `/#storage-driver-options`) ? The extra slash is added by some JavaScript library, but is not the _actual_ anchor

File: daemon/graphdriver/devmapper/deviceset.go [hunk 2]
```diff
@@ -1706,9 +1706,9 @@ func (devices *DeviceSet) initDevmapper(doInit bool) error {
...
 		} else {
-			logrus.Error("devmapper: Udev sync is not supported. This will lead to data loss and unexpected behavior. Install a more recent version of libdevmapper or select a different storage driver. For more information, see https://docs.docker.com/engine/reference/commandline/daemon/#daemon-storage-driver-option")
+			logrus.Error("devmapper: Udev sync is not supported. This will lead to data loss and unexpected behavior. Install a more recent version of libdevmapper or select a different storage driver. For more information, see https://docs.docker.com/engine/reference/commandline/dockerd/#/storage-driver-options")
```
Comment (thaJeztah) [inline daemon/graphdriver/devmapper/deviceset.go]: Same here

**Review by thaJeztah — CHANGES_REQUESTED** [review]

**Comment by tonistiigi** [comment]
@thaJeztah PTAL

**Review by thaJeztah — APPROVED** [review]
LGTM

**Comment by thaJeztah** [comment]
ping @vdemeester

**Review by vdemeester — APPROVED** [review]
LGTM 🐸

**Pull request merged**